scripts/build-win.py

```diff
@@ -30,13 +30,14 @@
     'rtc_use_h265=true',
     'is_component_build=false',
     'use_lld=false',
+    'use_rtti=true',
```

Comment:
RTTI is expected to be disabled. Is there any reason to enable it?

Reply:
MSDK util headers that C++ SDK on windows/Linux depends on turns on rtti.  Currently I workaround this by replacing all dynamic_cast<T> calls with reinterpret_cast<T> calls.

Further with switching to oneVPL those dynamic cast will be in oneVPL implementation itself. I'm not sure if this will cause problem if I'm linking to it without rtti turned on.

Reply:
Since this is a seperate change differs with the target of this PR, will revert this and submit seperate PR if neccessary.